api/types/plugin.go

```diff
@@ -111,6 +113,25 @@ func (p *PluginV1) CheckAndSetDefaults() error {
 		if (bearer.Token == "") == (bearer.TokenFile == "") {
 			return trace.BadParameter("exactly one of Token and TokenFile must be specified")
 		}
+	case *PluginSpecV1_Okta:
+		// Check settings.
+		if settings.Okta == nil {
+			return trace.BadParameter("settings must be set")
```

Comment:
```suggestion
			return trace.BadParameter("missing Okta settings")
```
nit: I think adding a little more info to the error would be helpful